xml/System.IO.Enumeration/FileSystemEnumerable`1.xml

```diff
@@ -54,10 +54,10 @@
         <Parameter Name="options" Type="System.IO.EnumerationOptions" />
       </Parameters>
       <Docs>
-        <param name="directory">To be added.</param>
-        <param name="transform">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="directory">The path of the directory where the enumeration will be performed.</param>
```

Comment:
do we need to say whether this is a full or relative path?

Reply:
It doesn't matter. I followed the code path this constructor takes, and we eventually try to normalize the path / get the full path here: https://source.dot.net/#System.IO.FileSystem/System/IO/Enumeration/FileSystemEnumerator.cs,38